staging/src/k8s.io/kube-scheduler/extender/v1/types.go

```diff
@@ -92,6 +92,10 @@ type ExtenderFilterResult struct {
 	NodeNames *[]string
 	// Filtered out nodes where the pod can't be scheduled and the failure messages
 	FailedNodes FailedNodesMap
+	// Filtered out nodes where the pod can't be scheduled and preemption would
+	// not change anything. The value is the failure message same as FailedNodes.
+	// Nodes specified here takes precedence over FailedNodes.
+	FailedAndUnresolvableNodes FailedNodesMap
```

Comment:
is it an error to include a node in FailedNodes and not in this map?

Reply:
> ...and preemption would not change anything

is that what unresolveable means? the doc doesn't seem to match the variable name

Reply:
> is it an error to include a node in FailedNodes and not in this map?

No. We are practically considering the union, however nodes in this map don't get considered during preemption.

> > ...and preemption would not change anything
> 
> is that what unresolveable means? the doc doesn't seem to match the variable name

Unresolvable is the term used in the scheduler pkg to indicate that preempting pods from a node doesn't make it schedulable for an incoming pod.

Reply:
I see. I parsed this as "(Failed Nodes) And (Unresolveable Nodes)" instead of "(Failed And Unresolveable) Nodes". If Unresolveable implies Failed, then would it be clearer to just call this UnresolveableNodes?

Reply:
this tries to mimic scheduling plugin statuses, which could be "Unschedulable" or "UnschedulableAndUnresolvable".

but I don't have a strong opinion about removing the "Failed" part

Reply:
if this is following an existing pattern in this API, that's fine